src/main/flight/pid.c

```diff
@@ -404,17 +409,20 @@ void pidInitFilters(const pidProfile_t *pidProfile)
         }
     }
 #endif
-#if defined(USE_D_CUT)
-    if (pidProfile->dterm_cut_percent == 0) {
-        dtermCutRangeApplyFn = nullFilterApply;
-        dtermCutLowpassApplyFn = nullFilterApply;
-    } else {
-        dtermCutRangeApplyFn = (filterApplyFnPtr)biquadFilterApply;
-        dtermCutLowpassApplyFn = (filterApplyFnPtr)pt1FilterApply;
-        for (int axis = FD_ROLL; axis <= FD_YAW; axis++) {
-            biquadFilterInitLPF(&dtermCutRange[axis], pidProfile->dterm_cut_range_hz, targetPidLooptime);
-            pt1FilterInit(&dtermCutLowpass[axis], pt1FilterGain(pidProfile->dterm_cut_lowpass_hz, dT));
-        }
+#if defined(USE_D_MIN)
+    dMin[FD_ROLL] = pidProfile->d_min_roll;
+    dMin[FD_PITCH] = pidProfile->d_min_pitch;
+    dMin[FD_YAW] = pidProfile->d_min_yaw;
+    for (int axis = FD_ROLL; axis <= FD_YAW; axis++) {
+        if (dMin[axis] == 0) {
+            dMinRangeApplyFn = nullFilterApply;
+            dMinLowpassApplyFn = nullFilterApply;
+        } else {
+            dMinRangeApplyFn = (filterApplyFnPtr)biquadFilterApply;
+            dMinLowpassApplyFn = (filterApplyFnPtr)pt1FilterApply;
+
+            biquadFilterInitLPF(&dMinRange[axis], D_MIN_RANGE_HZ, targetPidLooptime);
+            pt1FilterInit(&dMinLowpass[axis], pt1FilterGain(D_MIN_LOWPASS_HZ, dT));        }
```

Comment:
the `}` on this line needs to go down one

Reply:
done